src/components/BookList/AddBook.js

```diff
@@ -40,24 +41,26 @@ function AddBook() {
         />
         <select
           name="author"
+          className="dropdown"
           value={bookAuthor}
           onChange={(e) => setAuthor(e.target.value)}
         >
-          <option value="">-- Select Author --</option>
+          <option value=""> Author </option>
           <option value="Suzzane Collins">Suzzane Collins</option>
           <option value="Frank Hebert">Frank Hebert</option>
           <option value="Robert Greene">Robert Greene</option>
         </select>
         <select
           name="category"
+          className="dropdown"
           value={bookCategory}
           onChange={(e) => setCategory(e.target.value)}
         >
-          <option value="">-- Select Category --</option>
+          <option value="">Select Category </option>
           <option value="Fiction">Fiction</option>
           <option value="Nonfiction">Nonfiction</option>
         </select>
-        <button type="submit" disabled={checkStatus === 'loading'}>ADD BOOK</button>
+        <button type="submit" disabled={checkStatus === 'loading'} className="add-book">ADD BOOK</button>
```

Comment:
Nice job styling your project to match the given design😉. Let's try to add some changes to our form section to make it match the given design😉. Currently, I noticed your form section is causing the page to overflow and this is because the elements are too much to fit in the width of the screen.
<img width="1440" alt="image" src="https://github.com/migo1/book-store/assets/64643995/7694fef9-34cf-464c-97c0-a390e3ad02da">

from the design, there's no need for the category button since we are not giving any action to it😉. Hence, I suggest you remove the category dropdown and style the section to match the design😉. don't forget to consider the width of the form element and the font properties used😉.

design
<img width="812" alt="image" src="https://github.com/migo1/book-store/assets/64643995/19962e42-df4e-4d6a-a64f-732e142a0fb4">

Reply:
![Screenshot from 2023-07-21 16-57-09](https://github.com/migo1/book-store/assets/31877143/9756f62e-70d4-43cd-af96-97fa497701dd)

the elements are ok